src/skin.js

```diff
@@ -21,3 +21,10 @@ export function unuseInline () {
   skin.unuse()
   inline.unuse()
 }
+
+export default {
```

Comment:
We shouldn't rely on object literal short hand here since it doesn't get transpiled, but otherwise LGTM.